.github/workflows/pythonpackage.yml

```diff
@@ -6,33 +6,36 @@ jobs:
   build:
     runs-on: ${{ matrix.os }}
     strategy:
+      fail-fast: false
```

Comment:
Why disable fail fast? I'd rather not have to wait for everything to complete if there is an error.

Reply:
You would still get instant feedback for each individual matrix job, but in a case where the tests will fail for Windows on python 3.6 and 3.7, but not 3.8 or 3.9, this will give you the success/failure for every other job, while `fail-fast true` would mean you only get feedback on the first failed job, then every other job would be aborted. Hope that makes sense :) 

Removing it for now 👍